internal/pkg/eql/visitor.go

```diff
@@ -260,6 +264,25 @@ func (v *expVisitor) VisitVariableExp(ctx *parser.VariableExpContext) interface{
 			return resolved
 		}
 	}
+	if !v.allowMissingVars {
+		names := []string{}
+		for _, entry := range ctx.AllVariable() {
+			// Wrap the name in quotes for use in error
+			names = append(names, fmt.Sprintf("\"%v\"", entry.GetText()))
```

Comment:
Nit: alternative method of wrapping in quotes:
```suggestion
			names = append(names, fmt.Sprintf("%q", entry.GetText()))
```